src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -400,9 +438,11 @@ private void TriggerStatusLoader()
             }
         }
 
-        private async System.Threading.Tasks.Task ReloadPackageVersionsAsync()
+        public async System.Threading.Tasks.Task ReloadPackageVersionsAsync()
         {
+            IncrementTask();
             var result = await _backgroundLatestVersionLoader.Value;
+            DecrementTask();
```

Comment:
Look at `Interlocked`'s `Increment(...)`, `Decrement(...)`, and `CompareExchange(...)`.

Reply:
Sure - I think that'll be an easy refactoring. Had seen but avoided Increment/Decrement due to my use of a property (but I can just fire OnPropertyChanged). Didn't know about CompareExchange.

Is this just a refactoring suggestion, or do you already see something it'll improve?